Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -290,7 +294,9 @@ void doEsirkepovDepositionShapeN (const GetParticlePosition& GetPosition,
     // (do_ionization=1)
     bool const do_ionization = ion_lev;
     Real const dxi = 1.0_rt / dx[0];
+#if !(defined WARPX_DIM_RZ)
     Real const dtsdx0 = dt*dxi;
+#endif
```

Comment:
Fixes the warning:
`./Source/Particles/Deposition/CurrentDeposition.H:295:16`: warning: unused variable ‘dtsdx0’ [-Wunused-variable]
     Real const dtsdx0 = dt*dxi;